src/helpers.js

```diff
@@ -208,8 +208,9 @@ const generateInvalidTrace = async (
  */
 export async function processESLintMessages(response, textEditor, showRule, worker) {
   return Promise.all(response.map(async ({
-    message, line, severity, ruleId, column, fix, endLine, endColumn
+    fatal, message: originalMessage, line, severity, ruleId, column, fix, endLine, endColumn
   }) => {
+    const message = fatal ? originalMessage.split('\n').shift() : originalMessage
```

Comment:
Since the Array being created here is being discarded within the same command, just do `[0]` on the end, it's clearer what the intent is.